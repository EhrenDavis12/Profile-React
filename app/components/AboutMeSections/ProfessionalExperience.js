import React from 'react';
import Banner from 'components/Banners/Banner';
import AttainiaExperience from 'components/Professional/AttainiaExperience';
import HylandExperience from 'components/Professional/HylandExperience';
import CadtelExperience from 'components/Professional/CadtelExperience';

// const BGIcon = require('images/BGIcon.png');

function ProfessionalExperience() {
  return (
    <div className="container">
      <Banner>
        <AttainiaExperience />
      </Banner>

      <Banner>
        <HylandExperience />
      </Banner>

      <Banner>
        <CadtelExperience />
      </Banner>
    </div>
  );
}

export default ProfessionalExperience;
