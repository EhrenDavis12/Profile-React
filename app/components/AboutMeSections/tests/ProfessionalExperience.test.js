import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';
import ProfessionalExperience from '../ProfessionalExperience';

describe('<ProfessionalExperience />', () => {
  it('should render the page', () => {
    const renderedComponent = shallow(<ProfessionalExperience />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
