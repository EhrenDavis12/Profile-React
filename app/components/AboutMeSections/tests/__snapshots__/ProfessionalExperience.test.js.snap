// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfessionalExperience /> should render the page 1`] = `
<div
  className="container"
>
  <BannerSection>
    <AttainiaExperience />
  </BannerSection>
  <BannerSection>
    <HylandExperience />
  </BannerSection>
  <BannerSection>
    <CadtelExperience />
  </BannerSection>
</div>
`;
